website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -51,7 +51,7 @@ Notifications are sent as the run progresses, and can be triggered on one or mor
 | Created            | `"run:created"`         | A run is created and enters the [Pending stage](/cloud-docs/run/states#1-the-pending-stage)                                                    |
 | Planning           | `"run:planning"`        | A run acquires the lock and starts to execute.                                                      |
 | Needs Attention    | `"run:needs_attention"` | A plan has changes and Terraform requires user input to continue. This may include approving the plan or a [policy override](/cloud-docs/run/states#7-the-policy-check-stage). |
```

Comment:
```suggestion
| Needs Attention    | `"run:needs_attention"` | A plan has changes and Terraform requires user input to continue. This input may include approving the plan or a [policy override](/cloud-docs/run/states#7-the-policy-check-stage). |
```

Suggest adding "input" after "This" to avoid a "This" without a subject.